src/openforms/forms/tests/test_data_migrations.py

```diff
@@ -521,3 +527,62 @@ def test_open_when_empty_is_false(self):
             component = components["editgrid4"]
 
             self.assertFalse(component["openWhenEmpty"])
+
+
+class FormVariableMigrationTests(TestCase):
+    app = "forms"
+
+    def setUp(self):
+        self.form_definition = FormDefinition.objects.create(
+            name="FormDefinition1",
+            configuration={
+                "components": [
+                    {
+                        "key": "theFirstName",
+                        "type": "textfield",
+                        "label": "First Name",
+                        "prefill": {
+                            "plugin": "demo",
+                            "attribute": "",
+                        },
+                    },
+                ]
+            }
+        )
+        self.form = Form.objects.create(name="Form1")
+        self.formstep = FormStep.objects.create(
+            form=self.form, form_definition=self.form_definition
+        )
+        self.form_definition2 = FormDefinition.objects.create(
+            name="FormDefinition2",
+            configuration={
+                "components": [
+                    {
+                        "key": "theFirstName",
+                        "type": "textfield",
+                        "label": "First Name",
+                        "prefill": {
+                            "plugin": "",
+                            "attribute": "voornamen",
+                        },
+                    },
+                ]
+            }
+        )
+        self.form2 = Form.objects.create(name="Form2")
+        self.formstep2 = FormStep.objects.create(
+            form=self.form2, form_definition=self.form_definition2
+        )
```

Comment:
don't do this sort of indirection, do your data setup inside of your test function.

Also, when you _do_ implement `setUp`, _do_ call `super().setUp()` so that future added mixins work out of the box